data/weather-symbol-modes/intensity_categories.json

```diff
@@ -3,5 +3,7 @@
     "rain": [15, 12],
     "snow_shower": [26, 23],
     "snow": [27, 24],
-    "thunder": [30, 29]
+    "thunder": [30, 29],
+    "cloud": [7, 8],
```

Comment:
Is 'overcast' (8) a more intense option than 'cloudy' (7)? If it is, then should the order of this category be switched? The impact of the current order of this category is shown in the second change in the acceptance test data in your notebook (https://www-nwp/~bayliffe/wxmodal_acceptance_tests/). 

![image](https://github.com/user-attachments/assets/918457d1-6436-405f-9212-4273cd24e803)

Reply:
See https://github.com/metoppv/improver/pull/2050#discussion_r1860969891